src/components/NoteDetails/NoteDetailsComponent.js

```diff
@@ -1,19 +1,44 @@
 import React, { Component } from 'react';
-import { StyledNoteNameEditor } from 'components/NoteDetails/NoteDetailsStyles';
+import {
+  StyledNoteNameEditor,
+  StyledDetailsScreen,
+  StyledNoteContentEditor,
+} from 'components/NoteDetails/NoteDetailsStyles';
 
 export class NoteDetailsComponent extends Component {
-  handleChange(e) {
-    this.props.onNoteNameChange(e.target.value);
+  handleChangeNoteName(e) {
+    this.props.onNoteNameChange({
+      newNoteName: e.target.value,
+      note: this.props.selectedNote,
+    });
+  }
+
+  handleChangeNoteContent(e) {
+    this.props.onNoteContentChange({
+      note: this.props.selectedNote,
+      noteText: e.target.value,
+    });
   }
 
   render() {
     return (
-      <StyledNoteNameEditor
-        onChange={e => {
-          this.handleChange(e);
-        }}
-        value={this.props.noteName}
-      />
+      <StyledDetailsScreen>
```

Comment:
I think it should be `StyledNoteDetailsScreen`, since it contains `StyledNoteNameEditor`, `StyledNoteContentEditor`. So to be consistent in naming: either we omit `Note` in all the names, or we add `Note` to all the names.

I think it's better to have `Note` in the name, because "details screen" is a very general name: details of what? `NoteDetails` makes the component name much more descriptive: this is for note details.